webapp/src/features/adminEditOrganization/AdminEditOrganization.tsx

```diff
@@ -35,6 +37,12 @@ const AdminEditOrganization = () => {
     },
   })
 
+  const { orgName } = useParams<{ orgName: string }>()
+
+  useEffect(() => {
+    getOrgData({ orgName })
```

Comment:
We'll want a check for editing non-existent orgs. This edit page also doesn't seem to work, I'm unable to edit an org name.

Reply:
Added a check!

Reply:
Fixed the issue of orgs not re-naming as well